adapters/InMobi/InMobiAdapter/GADMAdapterInMobi.m

```diff
@@ -6,43 +6,13 @@
 
 #import "GADMAdapterInMobi.h"
 
-#import <InMobiSDK/IMSdk.h>
-
 #import "GADInMobiExtras.h"
 #import "GADMAdapterInMobiConstants.h"
-#import "GADMAdapterInMobiInitializer.h"
-#import "GADMAdapterInMobiUnifiedNativeAd.h"
-#import "GADMAdapterInMobiUtils.h"
-#import "GADMInMobiConsent.h"
 #import "GADMediationAdapterInMobi.h"
-#import "NativeAdKeys.h"
-
-/// Find closest supported ad size from a given ad size.
-static CGSize GADMAdapterInMobiSupportedAdSizeFromGADAdSize(GADAdSize gadAdSize) {
-  // Supported sizes
-  // 320 x 50
-  // 300 x 250
-  // 728 x 90
-
-  NSArray<NSValue *> *potentialSizeValues =
-      @[ @(GADAdSizeBanner), @(GADAdSizeMediumRectangle), @(GADAdSizeLeaderboard) ];
 
-  GADAdSize closestSize = GADClosestValidSizeForAdSizes(gadAdSize, potentialSizeValues);
-  return CGSizeFromGADAdSize(closestSize);
-}
 
 @implementation GADMAdapterInMobi {
-  /// Google Mobile Ads SDK ad network connector.
-  __weak id<GADMAdNetworkConnector> _connector;
-
-  /// InMobi banner ad object.
-  IMBanner *_adView;
-
-  /// InMobi interstitial ad object.
-  IMInterstitial *_interstitial;
 
-  /// Google Mobile Ads unified native ad wrapper.
-  GADMAdapterInMobiUnifiedNativeAd *_nativeAd;
 }
 
```

Comment:
Can remove all the methods from [`GADMAdNetworkAdapter`](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMAdNetworkAdapter) protocol.